lte/gateway/c/oai/tasks/sgw/sgw_handlers.c

```diff
@@ -2143,9 +2144,23 @@ int sgw_handle_nw_initiated_actv_bearer_req(
     "Received Dedicated Bearer Req Activation from PGW for LBI %d\n",
     itti_s5_actv_bearer_req->lbi);
 
-  /*Create dedicated bearer context-Bearer Context will be created
-   * after receiving response from MME
-  */
+  s_plus_p_gw_eps_bearer_context_information_t*
+    s_plus_p_gw_eps_bearer_ctxt_info_p = NULL;
+  hashtable_rc_t hash_rc = HASH_TABLE_OK;
+
+  hash_rc = hashtable_ts_get(
+    state->sgw_state.s11_bearer_context_information,
+    itti_s5_actv_bearer_req->s_gw_teid_S11_S4,
+    (void**) &s_plus_p_gw_eps_bearer_ctxt_info_p);
+
+  if (HASH_TABLE_OK != hash_rc) {
+    OAILOG_ERROR(
+      LOG_SPGW_APP,
+      "Did not find hash table entry for teid %u"
+      "for S11_NW_INITIATED_BEARER_ACTV_REQUEST\n",
+      itti_s5_actv_bearer_req->s_gw_teid_S11_S4);
+    OAILOG_FUNC_RETURN(LOG_SPGW_APP, rc);
```

Comment:
Why do we return RETURNok here? Should not we send back a reject message?

Reply:
Changed to RETRUNerror. Caller function takes care of sending reject if return type is RETURNerror